1.5.0

```diff
@@ -550,6 +550,25 @@
           }
         }
       ]
+    },
+    {
+      "instructionTypeName": "AssignPermissionSet",
```

Comment:
You shouldn't need to use instructionTypeName unless you have to specify a page layout.